### ISSUE TITLE
Is there any roadmap?

### ISSUE THREAD

**Issue by zhanglistar** [facebookincubator/velox#2050]
Thanks in advance.

**Comment by dingqiaoyi** [comment]
I'm also interested in when the GA version will be released. 
Is it likely to be released this year？@pedroerp @mbasmanova

**Comment by mbasmanova** [comment]
@dingqiaoyi We do not have a publicly sharable roadmap. Some of the major in-progress items are:

- decimal support
- window function support
- new optimized Parquet reader
- integration with Presto and Spark

Is there something specific you are interested in?

CC: @majetideepak

**Comment by dingqiaoyi** [comment]
> @dingqiaoyi We do not have a publicly sharable roadmap. Some of the major in-progress items are:
> 
> * decimal support
> * window function support
> * new optimized Parquet reader
> * integration with Presto and Spark
> 
> Is there something specific you are interested in?
> 
> CC: @majetideepak

Is there an expected time for Velox fully supporting TPC-H and TPC-DS ?

**Comment by mbasmanova** [comment]
> Is there an expected time for Velox fully supporting TPC-H and TPC-DS ?

TPC-H is already supported with Presto. @rui-mo Rui, do we have it supported with Spark as well?

TPC-DS will take a few more months.

**Comment by rui-mo** [comment]
> > Is there an expected time for Velox fully supporting TPC-H and TPC-DS ?
> 
> TPC-H is already supported with Presto. @rui-mo Rui, do we have it supported with Spark as well?
> 
> TPC-DS will take a few more months.

Yes, we've supported TPC-H in Spark with most operators offloaded into Velox.

**Comment by zhanglistar** [comment]
Very glad to see the progress and we are very eager to use and improve performance of spark and presto now, and also we will contribute back if we have improvements.

**Comment by pedroerp** [comment]
@zhanglistar great to hear, we can most definitely use the help! 

Both Presto and Spark integrations are quickly maturing and can already run TPC-H (and most TPC-DS). We're now iterating on performance to ensure there are no unexpected bottlenecks. 

Moving this to a github Discussion.
